src/coreComponents/constitutive/fluid/PVTFunctions/EzrokhiBrineDensity.cpp

```diff
@@ -0,0 +1,263 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file EzrokhiBrineDensity.cpp
+ */
+
+#include "constitutive/fluid/PVTFunctions/EzrokhiBrineDensity.hpp"
+
+#include "constitutive/fluid/PVTFunctions/PVTFunctionHelpers.hpp"
+#include "functions/FunctionManager.hpp"
+
+namespace geosx
+{
+
+using namespace stringutilities;
+
+namespace constitutive
+{
+
+namespace PVTProps
+{
+
+namespace
+{
+
+TableFunction const * makeSatDensityTable( string const & functionName,
+                                           FunctionManager & functionManager )
+{
+  array1d< array1d< real64 > > temperatures;
+  array1d< real64 > densities;
+
+  temperatures.resize( 1 );
+  temperatures[0].resize( 26 );
+  densities.resize( 26 );
+
+  temperatures[0][0] = 0.01;
+  temperatures[0][1] = 10;
+  temperatures[0][2] = 20;
+  temperatures[0][3] = 25;
+  temperatures[0][4] = 30;
+  temperatures[0][5] = 40;
+  temperatures[0][6] = 50;
+  temperatures[0][7] = 60;
+  temperatures[0][8] = 70;
+  temperatures[0][9] = 80;
+  temperatures[0][10] = 90;
+  temperatures[0][11] = 100;
+  temperatures[0][12] = 110;
+  temperatures[0][13] = 120;
+  temperatures[0][14] = 140;
+  temperatures[0][15] = 160;
+  temperatures[0][16] = 180;
+  temperatures[0][17] = 200;
+  temperatures[0][18] = 220;
+  temperatures[0][19] = 240;
+  temperatures[0][20] = 260;
+  temperatures[0][21] = 280;
+  temperatures[0][22] = 300;
+  temperatures[0][23] = 320;
+  temperatures[0][24] = 340;
+  temperatures[0][25] = 360;
+
+
+  densities[0] = 999.85;
+  densities[1] = 999.7;
+  densities[2] = 998.21;
+  densities[3] = 997.05;
+  densities[4] = 995.65;
+  densities[5] = 995.65;
+  densities[6] = 988.04;
+  densities[7] = 983.2;
+  densities[8] = 977.76;
+  densities[9] = 971.79;
+  densities[10] = 965.31;
+  densities[11] = 958.35; // this value is used as referense density with referense pressure 100 kPa.
```

Comment:
```suggestion
  densities[11] = 958.35; // this value is used as reference density with reference pressure 100 kPa.
```

Reply:
Thanks for looking into it @CusiniM!
Actually, this whole comment is irrelevant after b9535f7bdf92304b9ae476ec90599da9b1d8d9da
I`m removing it.